packages/nuxt/src/module.ts

```diff
@@ -85,3 +100,53 @@ export default defineNuxtModule<ModuleOptions>({
     });
   },
 });
+
+async function injectStyles(
+  moduleOptions: ModuleOptions,
+  nuxt: Nuxt,
+  resolver: Resolver,
+) {
+  /**
+   * Inject UploadThing stylesheet if no Tailwind is installed
+   */
+  if (!hasNuxtModule("@nuxtjs/tailwindcss", nuxt)) {
+    nuxt.options.css.push("@uploadthing/vue/styles.css");
+    return;
+  }
+
+  /**
+   * Else we install our tailwind plugin
+   */
+
+  const vueDist = await resolver.resolvePath("@uploadthing/vue");
+  const contentPath = dirname(vueDist) + sep + "**";
+
+  const template = addTemplate({
+    filename: "uploadthing.tailwind.config.cjs",
+    write: true,
+    getContents: () => `
+      const { uploadthingPlugin } = require('uploadthing/tw');
+
+      module.exports = {
+        content: [
+          ${JSON.stringify(contentPath)}
+        ],
+        plugins: [
+          require('uploadthing/tw').uploadthingPlugin
+        ]
+      }
+    `,
+  });
+
+  // @ts-expect-error - Help pls
```

Comment:
you can probably get the types in your workspace by creating a `packages/nuxt/.nuxtrc` file with `@nuxtjs/tailwindcss` in it

Reply:
hmm we already have a nuxtrc but adding `modules[]='@nuxtjs/tailwindcss'` didn't seem do anything

Reply:
you'll also need to rerun the `dev:prepare` step

Reply:
yea it looks like it added it but `options` type still doens't have `tailwindcss` on it, even after reinstalling restarting ts server

![CleanShot 2024-11-04 at 23 39 11@2x](https://github.com/user-attachments/assets/6b74ad38-a613-441f-91f7-13298faac3f9)
![CleanShot 2024-11-04 at 23 40 05@2x](https://github.com/user-attachments/assets/a67d200b-c023-4a13-825e-15c6162f0e30)
![CleanShot 2024-11-04 at 23 40 48@2x](https://github.com/user-attachments/assets/b6e2a175-1352-4c3b-9907-3b313ce7dd4f)

Reply:
could do this but not sure how much better that is xd 

![CleanShot 2024-11-04 at 23 48 48@2x](https://github.com/user-attachments/assets/30da8b59-c915-4b9a-a2e4-a3bee3769dd2)

Reply:
I guess this is a bit better [`f913eb5` (#1033)](https://github.com/pingdotgg/uploadthing/pull/1033/commits/f913eb537570bba904c2c2b5c330d68393fcd398)